app/uk/gov/hmrc/trusts/controllers/EstateBaseController.scala

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.trusts.controllers
+
+import play.api.data.validation.ValidationError
+import play.api.libs.json._
+import play.api.mvc.{Request, Result}
+import uk.gov.hmrc.play.bootstrap.controller.BaseController
+import uk.gov.hmrc.trusts.utils.ErrorResponses._
+
+import scala.concurrent.Future
+
+class EstateBaseController extends BaseController {
```

Comment:
I don't think we need to create this EstateBaseController. We're duplicating a lot of functionality that is the same across estates and trusts.

Reply:
removed esatesbasecontroller and trustbasecontroller, merged them in to variationsbasecontroller